constraint_utils/constraint_basics.py

```diff
@@ -378,6 +378,29 @@ def keep_criteria(ht):
 
 def build_models(coverage_ht: hl.Table, trimers: bool = False, weighted: bool = False, half_cutoff = False,
                  ) -> Tuple[Tuple[float, float], Dict[str, Tuple[float, float]]]:
+    """
+    Build coverage model and plateau models.
+    
```

Comment:
As I mentioned in my overall comment from the last round of review, I still think this needs more description of what models are being built. 

"""I think you need more info in the docstring to explain what the function is doing. For example how is the high coverage cutoff actually used? I think reading and borrowing from Konrad's supplement (the Constraint modeling section) can help you with this. The idea is that we want someone to fully understand what the function is doing without needing to completely read through the code."""